api/src/main/java/org/pmiops/workbench/api/OfflineWorkspaceController.java

```diff
@@ -1,70 +0,0 @@
-package org.pmiops.workbench.api;
-
-import java.time.LocalDate;
-import java.time.LocalDateTime;
-import java.util.List;
-import java.util.stream.Collectors;
-import javax.inject.Provider;
-import org.pmiops.workbench.config.WorkbenchConfig;
-import org.pmiops.workbench.db.dao.WorkspaceDao;
-import org.pmiops.workbench.db.model.DbWorkspace;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.RestController;
-
-@RestController
-public class OfflineWorkspaceController implements OfflineWorkspaceApiDelegate {
```

Comment:
"offline" controllers are only called by crons, not the API.  So it's safe to delete them without regard to UI deployment.